bin/compile

```diff
@@ -361,6 +361,11 @@ if [ "$STACK" = "scalingo" ] ; then
     test ! -f "bin/tail" && ln -s "/app/vendor/coreutils/bin/tail" "bin/tail"
 fi
 
+# If this buildpack is run after the nodejs buildpack remove the WEB_CONCURRENCY script
```

Comment:
Why not overriding WEB_CONCURRENCY.sh with PHP custom script to configure this variable?

Reply:
Because in the PHP buildpack it's bin/run that compute the WEB_CONCURRENCY. I don't see the point of moving this logic out.

Reply:
OK